app/process/watcher/message/watcher.go

```diff
@@ -60,22 +60,29 @@ func NewWatcher(client client.MirrorNode, topicID string, repository repository.
 }
 
 func (cmw Watcher) Watch(q *pair.Queue) {
+	if !cmw.client.TopicExists(cmw.topicID) {
+		cmw.logger.Errorf("Error incoming: Could not start monitoring topic [%s] - Topic not found.", cmw.topicID.String())
```

Comment:
In the error logs if we are using `Errorf` it is indicated that it is error. I think that there is no need to further emphasise the error by having "Error incoming"